task_DEV_4/task_DEV_4/Square.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace task_DEV_4
+{
+    class Square
+    {
+        public double Side { get; set; }
+
+        public Square(double side)
+        {
+            Side = side;
+        }
+
+        public void SetSquareParameter(double side)
```

Comment:
I guess it is not necessary to create this method only for one parameter because we have already property Side that has setter. It can be removed.

Reply:
Done